assets/airports/loww.json

```diff
@@ -339,6 +590,152 @@
       "draw": [["WW152", "WGM"], ["WW123", "WGM"], ["WW160", "WGM"], ["WW304", "WGM"]]
     }
   },
+  "stars": {
+    "NERDU4K": {
+     "icao": "NERDU4K",
+     "name": "Nerdu four kilo",
+     "transitions": {
+          "NERDU": ["NERDU"]
+      },
+     "body": [["NERDU", "A80|S250"], ["WW666", "A80-|S250"], ["WW664", "A50+|S250"], ["WW658", "A50|S250"], ["WW656", "A50-|S250"], ["WW654", "A50-|S250"], ["WW653", "A50-|S250"], ["WW652", "A50-|S250"], ["WW651", "A50-|S250"], ["WW650", "A50-|S250"]],
+     "draw": ["NERDU", "WW666", "WW664", "WW658", "WW656", "WW654", "WW653", "WW652", "WW651", "WW650"]
+    },
+    "NERDU4M": {
+     "icao": "NERDU4M",
+     "name": "Nerdu four mike",
+     "transitions": {
+          "NERDU": ["NERDU"]
+      },
+     "body": [["NERDU", "A60+|S250"], ["WW963", "A70+|S250"], ["WW961", "A70|S250"], ["WW959", "A60|S250"], ["WW957", "A60-|S250"], ["WW955", "A60-|S250"], ["WW953", "A60-|S250"], ["WW952", "A30|S250"], ["WW951", "A30-|S250"], ["WW950", "A30-|S250"]],
+     "draw": [["NERDU", "A60+|S250"], ["WW963", "A60+|S250"], ["WW961", "A60+|S250"], ["WW959", "A60+|S250"], ["WW957", "A60+|S250"], ["WW955", "A60+|S250"], ["WW953", "A60+|S250"], ["WW952", "A30+|S250"], ["WW951", "A30+|S250"], ["WW950", "A30|S250"]]
```

Comment:
@jakobeng1303 the `draw` sections can't contain the same stuff that `body` does. They should just be an array of lines to draw from point to point. It's explained more in depth [here](http://zlsa.github.io/atc/documentation/airport-format.html).